### PR TITLE
Multiple keys in sidebar and simple search

### PR DESCRIPTION
**Pull request by Signez**
searchallthethings

### CODE REVIEW

File: src/models.rs [hunk 1]
```diff
@@ -3,14 +3,14 @@ pub struct Translation {
     pub id: i32,
     pub key: String,
     pub locale: String,
-    pub content: String,
+    pub content: Option<String>,
```
Comment (rlustin) [inline src/models.rs]: :+1:

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,7 @@
 database.sqlite
 /target/
 /src/frontend/node_modules/
+/src/frontend/public/
```
Comment (rlustin) [inline .gitignore]: Ouf !

File: tools/quick-import.rb [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+end
+
+translations_added = 0
+
+yaml.map do |key, values|
+  lang = key
+
+  nested_translation_groups = values
+
+  def flatten_hash(my_hash, parent=[])
```
Comment (rlustin) [inline tools/quick-import.rb]: Spaces around `=`

File: .gitignore (hunk 2 above)
Comment (rlustin) [inline .gitignore]: As you removed this directory, could you please remove the phrase that says it is committed on purpose from `README.md`?

File: tools/quick-import.rb [hunk 4]
```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env ruby
+
+if ARGV.length != 2
+  puts "usage: ./quick-import.rb <database-file.sqlite> <lang.yml>"
```
Comment (rlustin) [inline tools/quick-import.rb]: Could you also update `README.md` with instructions how to use this tool?

Comment (rlustin) [inline tools/quick-import.rb]: And maybe a link to a file we can use for development?

File: .gitignore (hunk 2 above)
Comment (Signez) [inline .gitignore]: Done :+1:

File: tools/quick-import.rb (hunk 4 above)
Comment (Signez) [inline tools/quick-import.rb]: Honestly, I don't think that this script really need more documentation for now, as it is mainly a hack useful for bootstrapping the environment. It is mainly self documented, and I hope we will be able to remove it quickly by replacing it by its Rust equivalent directly in Lugh :wink:

Comment (rlustin) [inline tools/quick-import.rb]: Sounds great! 👍

**Pull request merged**